### DISCUSSION TITLE
Table in the Schedule Editor does not show

### DISCUSSION THREAD

**Discussion by duongthuyle** [pretalx/pretalx#1917] [Q&A]
The table in the schedule editor does not show in my Pretalx instance. Did I do anything wrong during the installation or is there any missing? Please check the attached and if you have a solution, please help me.

<img width="1437" alt="Screenshot 2024-12-08 at 00 24 49" src="https://github.com/user-attachments/assets/95ce37f9-92da-49ee-ab41-963e80322e85">
<img width="1437" alt="Screenshot 2024-12-08 at 00 24 14" src="https://github.com/user-attachments/assets/75f194c9-9d88-4801-86e7-6be22e0a280f">
<img width="1437" alt="Screenshot 2024-12-08 at 00 24 05" src="https://github.com/user-attachments/assets/ee9a78b4-216e-4162-bca6-074ae03730cb">

**Comment by duongthuyle** [comment]
Other screenshots:

![z6064172803284_ee75ecc5f46b22ab79edf01528a75397](https://github.com/user-attachments/assets/74a1737c-c7de-4e3b-95fe-b5b30bb353d9)

Here is what it should look like:

![z6064182170875_8747141ad570cf9d37ab60c966454c39](https://github.com/user-attachments/assets/b0dd7a38-d3ba-41c4-85af-2d998b530b16)

**Comment by rixx** [comment]
Did you run `rebuild`? Can you show the browser console (network tab) to see if the static files are just not being loaded, and the browser console (console tab) to see error message from the JavaScript? This could be anything from misconfigured staticfiles (unlikely, as your CSS seems to be working) to an outdated browser, to HTTP security headers being misconfigured.
